src/sentry/static/sentry/app/components/sidebar/broadcasts.tsx

```diff
@@ -141,6 +142,7 @@ class Broadcasts extends React.Component<Props, State> {
             title={t("What's new in Sentry")}
             hidePanel={hidePanel}
           >
+            <BroadcastSdkUpdates />
```

Comment:
If the sidebar is collapsed will there be any indication that there are sdk updates available?

Reply:
I am still considering how to indicate that there are updates, since right now we don't have a decent way of indicating if a user has seen an SDK update list, an when we would consider them not to be unseen again.

Reply:
I am going to avoid indication for now. The indication will be driven by the dismissible alerts that I'm adding in a follow up PR throughout the app.